zman-backend/src/main/java/com/peterservice/zman/api/data/Authentication.kt

```diff
@@ -0,0 +1,3 @@
+package com.peterservice.zman.api.data
+
+data class Authentication (val userName: String, val isAuthenticated: Boolean)
```

Comment:
I'd set default "" for username. You won't have to pass empty username in Controller then